tests/cases/model_test.py

```diff
@@ -64,6 +65,96 @@ def setUp(self):
         ModelImporter.registerModel('fake_ac', FakeAcModel())
         ModelImporter.registerModel('fake', FakeModel())
 
+    def testProjectionUtils(self):
+        inclusionProjDict = {
+            'public': True,
+            'access': True,
+            'email': True,
+            'login': True
+        }
+        inclusionProjList = ['public', 'access', 'email', 'login']
+        exclusionProjDict = {
+            'public': False,
+            'access': False,
+            'email': False,
+            'login': False
+        }
+        overrideFields = ['access', 'public']
+
+        retval = _overwriteFields(inclusionProjDict, overrideFields)
+        self.assertEqual(retval, inclusionProjDict)
+        retval = _overwriteFields(inclusionProjList, overrideFields)
+        self.assertEqual(retval, inclusionProjList)
+        retval = _overwriteFields(exclusionProjDict, ['newValue'])
+        self.assertEqual(retval, exclusionProjDict)
+        retval = _overwriteFields(inclusionProjDict, ['newValue'])
+        self.assertEqual(retval, {
```

Comment:
Let's test the fact that `retval` should not be mutated by `_overwriteFields`.

Reply:
`retval` is never passed to `_overwriteFields`? I'm not really sure what that would be testing against?

Or are you saying the first argument, in this case `inclusionProjDict` should not be mutated by `_overwriteFields`?